README.md

```diff
@@ -48,16 +48,17 @@ In order to do a swap you will have to start two comit nodes.
 
 1. Install `docker` & `docker-compose`
 2. Install `node` (check the version required in package.json) & `yarn`
-3. Install `cargo-make`: `cargo install cargo-make`
-4. Run `cargo make` in the root folder of the repository, this will install various crates & tools such as rustfmt & clippy
+3. Install Rust nightly-2019-04-30: `rustup install nightly-2019-04-30` (this one is only used for `rustfmt`)
+4. Install `rustfmt` for `nightly-2019-04-30`: `rustup component add rustfmt --toolchain nightly-2019-04-30`
```

Comment:
📣 sometimes you put `nightly-2019-04-30` within backticks and sometimes not

Reply:
What I tried to do was having a _sentence_ that describes what you should do, followed by a command that you can copy and paste into the terminal.

Do you want me to change that.

Reply:
I am referring to point `3.` and `4.`

Reply:
Did you check the latest state if your comment is resolved now?